Upgraded to SSDK 15.1.0 and DBS 2022.10

ahrytsiuk opened the pull request.

ahrytsiuk commented:
@tarun-bb yes, I see.
```
Run VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
  VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
  VERSION_PARTS=($(echo $VERSION | tr "." "\n"))
  echo "MAJOR=${VERSION_PARTS[0]}" >> $GITHUB_ENV
  echo "MINOR=${VERSION_PARTS[1]}" >> $GITHUB_ENV
  echo "PATCH=${VERSION_PARTS[[2](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:2)]}" >> $GITHUB_ENV
  echo "DOCKER_TAG_VERSION=${VERSION}" >> $GITHUB_ENV
  shell: /usr/bin/bash -e {0}
  env:
    JAVA_HOME_17.0.5_x6[4](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:4): /opt/hostedtoolcache/jdk/17.0.[5](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:5)/x[6](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:6)4
    JAVA_HOME: /opt/hostedtoolcache/jdk/1[7](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:7).0.5/x64
    JAVA_HOME_17_0_5_X64: /opt/hostedtoolcache/jdk/17.0.5/x64
Error: Process completed with exit code 1.
```
But for me it's not clear what is the root cause. If you can give any hints how to fix a build - will be great!

tarun-bb commented:
> @tarun-bb yes, I see.
> 
> ```
> Run VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
>   VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
>   VERSION_PARTS=($(echo $VERSION | tr "." "\n"))
>   echo "MAJOR=${VERSION_PARTS[0]}" >> $GITHUB_ENV
>   echo "MINOR=${VERSION_PARTS[1]}" >> $GITHUB_ENV
>   echo "PATCH=${VERSION_PARTS[[2](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:2)]}" >> $GITHUB_ENV
>   echo "DOCKER_TAG_VERSION=${VERSION}" >> $GITHUB_ENV
>   shell: /usr/bin/bash -e {0}
>   env:
>     JAVA_HOME_17.0.5_x6[4](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:4): /opt/hostedtoolcache/jdk/17.0.[5](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:5)/x[6](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:6)4
>     JAVA_HOME: /opt/hostedtoolcache/jdk/1[7](https://github.com/Backbase/stream-services/actions/runs/3573122096/jobs/6008218235#step:7:7).0.5/x64
>     JAVA_HOME_17_0_5_X64: /opt/hostedtoolcache/jdk/17.0.5/x64
> Error: Process completed with exit code 1.
> ```
> 
> But for me it's not clear what is the root cause. If you can give any hints how to fix a build - will be great!

I think it's related to this issue https://backbase.slack.com/archives/CG2C5NH8D/p1669727698897109

uesleilima commented on .github/workflows/build.yml:
```diff
@@ -45,7 +45,7 @@ jobs:
 
       - name: Get version from POM
         run: |
-          VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
```
let's rollback this one as the problem was the JFrog downtime

ahrytsiuk commented:
After discussion with @uesleilima I pinned version for pinned spring-cloud-kubernetes to 2.1.13. Which was used in the Service SSDK 15.0.x
Between 2.1.13 and 2.1.14 there are no any new functionality is provided by the library. However, changes in the library cause tests to fail.

Additionally, I squashed comits and rebased branch based on latest changes from master.
